.github/pull_request_template.md

```diff
@@ -1,8 +1,34 @@
-# Technology Radar Pull Request Template
+<!--
+Hello 👋
+Thank you for contributing to our Technology Radar by proposing changes.
+Below you will find 2 different template sections:
+- [CONTENT] for PRs that propose changes to content
+- [CHORE] for PRs that propose changes to the infrastructure of the radar such as scripts, versioning, templates.
 
-When making a pull request, please ensure that the following
-things are done.
+🙏 Please use the relevant template section, it will help you to better communicate the proposed changes.
+⚠️ Be sure you remove the unused section
+-->
+<!-- SECTION: CONTENT -->
+<!--
+Required. Help reviewers with a hint on what this PR is about. Fill the table as per the following instructions:
```

Comment:
I'd remove this part. What we expect is to have PR that add content (markdown) which will be self-documenting.